.github/workflows/cd-syftcli.yml

```diff
@@ -30,6 +30,11 @@ jobs:
     if: always() && (needs.call-pr-tests-linting.result == 'success' || github.event.inputs.skip_tests == 'true')
     runs-on: ubuntu-latest
 
+    outputs:
+      deployed_version: ${{ steps.bump-version.outputs.deployed_version }}
```

Comment:
Q: Should the outputs, step, be defined, after the bump-version, step, if we define it before, maybe it would be null?

Reply:
Good question, it won't actually be null, defining it at the start is pretty standard. See here: https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs

Reply:
Oh I see, they are evaluated , at the end of each job.